lib/api/api_auth_webauthn.go

```diff
@@ -402,19 +373,23 @@ func (s *webauthnService) deleteOldStates() {
 	}
 }
 
-func newVolState() *WebauthnVolatileState {
-	s := WebauthnVolatileState{}
-	s.init()
+func newVolState() *apiproto.WebauthnVolatileState {
+	s := apiproto.WebauthnVolatileState{}
+	if s.Credentials == nil {
+		s.Credentials = make(map[string]*apiproto.WebauthnCredentialVolatileState, 1)
+	}
```

Comment:
Am I missing something here - what's that nil check for resp. why not initialise `Credentials` when definiting `s`.

Reply:
Ah, good catch! These two statements used to be in two separate functions. They were merged in f6269669b6ee6ed0f1a58c478013451b4ac47da6, but only naively.